### DISCUSSION TITLE
Security update to a non-stable version

### DISCUSSION THREAD

**Discussion by luizth** [SoftEtherVPN/SoftEtherVPN#2055] [Q&A] [answered]
I recently became aware of the security updated addressed in https://github.com/SoftEtherVPN/SoftEtherVPN/security/advisories/GHSA-j35p-p8pj-vqxq.

To the best of my knowledge, the security patch was released to a non-stable.

Since I run SoftEther VPN on a production environment, I am currently using the latest stable version 4.41.

Is it recommended that I update my VPN server to the latest version 5.02 to address the issue? Is the security patch being released to a stable version any time soon?

I'm sorry if I'm missing anything.

**Comment by chipitsine** [comment, marked as answer]
security impact is considered low. if you are afraid that your server will be used for 2-3 times amplification, you should be aware of such things.

after some time of testing in DE patch will be applied to Stable edition
